src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const LiElement = tree.querySelectorAll('li');
+
+LiElement.forEach((listItem) => {
+  const ulElement = listItem.querySelector('ul');
+
+  if (ulElement) {
+    const span = document.createElement('span');
+    const text = listItem.firstChild;
+
+    span.textContent = text.textContent;
+    listItem.insertBefore(span, text);
+    text.remove();
+  }
+});
+
+tree.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const nextSibling = e.target.nextSibling;
```

Comment:
The `nextSibling` property might not always be an element node. Consider checking if `nextSibling` is an element node before accessing `tagName`.